docs/bcfusion/validators/staking.md

```diff
@@ -0,0 +1,133 @@
+# Staking
+
+BNB smart chain (BSC) is a Proof-of-Staked-Authority (PoSA) blockchain, which means that staking is one of the most
+important parts of the system. [BEP-294](https://github.com/bnb-chain/BEPs/pull/294) introduces the new native staking
+mechanism after BNB chain fusion, which has several improvements:
+
+- Users are able to participate in staking on the BSC using BNB in BEP20 format.
```

Comment:
I think BNB is a native token, not a BEP20 format.

Reply:
fixed